eng/build.ps1

```diff
@@ -14,6 +14,12 @@
 # it's fine to call `build.ps1 -build -testDesktop` followed by repeated calls to
 # `.\build.ps1 -testDesktop`.
 
+if ($PSVersionTable.PSVersion.Major -lt "5") {
```

Comment:
I'd move it just before https://github.com/dotnet/roslyn/pull/33848/commits/947698d6b1bbe9b6597dd9421303968af6c7360e#diff-891b50b7d5fe7380fc246ed310019d61R69.

Reply:
Done